docs/kusion/4-configuration-walkthrough/4-workload.md

```diff
@@ -19,7 +15,7 @@ A `workload` maps to an `AppConfiguration` instance 1:1. If there are more than
     - [Create Files](#create-files)
     - [Customize container initialization](#customize-container-initialization)
 - [Configure Replicas](#configure-replicas)
-- [Differences between Services and Jobs](#differences-between-services-and-jobs)
+- [Differences between Services and Jobs](#differences-between-service-and-job)
```

Comment:
> Differences between Services and Jobs

Here seems to be different with the link title, maybe better to be `Differences between Service and Job`.